### DISCUSSION TITLE
Sequential load from multiple tables

### DISCUSSION THREAD

**Discussion by ma1ster** [dbt-labs/dbt-core#5419] [Q&A]
Hi,
I have a case where I need to load model from multiple tables, let's say 20. Union doesn't help me as tables are huge and would require deduplication on the go which further increase warehouse usage. I would like to load it one by one. Is it somehow possible in an elegant way ?

**Comment by p-hebert** [comment]
Hey @ma1ster!
Are you referring to the same use case as in my discussion [here](https://github.com/dbt-labs/dbt-core/discussions/5386)?

**Reply by ma1ster** [reply]
Hi @philippefutureboy , sorry for the late response. Yes I am looking for something similar
